exercises/practice/raindrops/.approaches/data-driven/content.md

```diff
@@ -0,0 +1,69 @@
+# Data-Driven Programming
+
+
```

Comment:
```suggestion

```

Reply:
Overall looks good to me, but I noticed the extra blank line which did not seem to be consistent.

Reply:
Thanks! Fixed.